Backpack-SwiftUI/SponsoredBanner/Classes/BPKSponsoredBanner.swift

```diff
@@ -170,10 +172,12 @@ public struct BPKSponsoredBanner: View {
             if let title = title {
                 BPKText(title, style: .label2)
                     .foregroundColor(variant.color)
+                    .lineLimit(20)
             }
             if let subheadline = subheadline {
                 BPKText(subheadline, style: .caption)
                     .foregroundColor(variant.color)
+                    .lineLimit(20)
```

Comment:
Can we consider using `nil` as a linelimit? With dynamic type we might need more lines. 20 feels like an arbitrary number as well.

Reply:
Did not know passing `nil` allowed unlimited lines! Modifying 👍